src/pacasam/connectors/lipac.py

```diff
@@ -90,14 +91,16 @@ def download_database(self, extraction_sql_query: str, max_chunksize_for_postgis
         gdf = gdf.set_crs(self.lambert_93_crs)
         gdf = gdf.sort_values(by=PATCH_ID_COLNAME)
         gdf = gdf.drop_duplicates(subset=PATCH_ID_COLNAME)
+        gdf[PATCH_ID_COLNAME] = gdf[[PATCH_ID_COLNAME, FILE_ID_COLNAME]].apply(
```

Comment:
not sure I understand this part: from what I understand,  you try to fix the patch id by transforming the one of the database to be able to use it in pacasam the same way as you would do with images patches. Isn't it easier (and cleaner) to fix the database directly so that this isn't required every time we use pacasam?

Reply:
By design, `VIGNETTE_ID` (that becomes `patch_id`) is a unique integer, that is created automatically everytime a patch is inserted in the database. So I guess we could, when the patch is inserted, create a new field that is also unique but contains the file ID as well. 
I am not sure this would be easier, since we then would have to query both ID (to join tables) and the new field PATCH_ID in the sql request, and then get rid of the former. Then we would also have to update the schema of LiPaC by adding a new field in one of the tables that was not designed to be updated, diverging from [the current ORM](https://github.com/IGNF/panini/blob/main/lipac_tables/tile.py)
@MichelDaab What do you think?

Reply:
I made the concatenation happen in the sql query directly, so we do not duplicate info in the database itself.